README.md

```diff
@@ -16,6 +16,23 @@ kubectl --context=$MY_CONTEXT -n $MY_NAMESPACE exec -it $POD_NAME bash
 kubectl --context=$MY_CONTEXT -n $MY_NAMESPACE delete deployment debugging-tools
 ```
 
+## SSH Server for reverse tunneling
+
+This repo also contains the Dockerfile for a ssh server, useful for making reverse tunnels to achieve a kind of "kubectl reverse-port-forward" behavior, and be able to expose services running in your local machine to the rest of the cluster. This server requires that you mount a `id_rsa.pub` key into `/root/.ssh/authorized_keys` to be able to log in.
+```bash
+MY_CONTEXT=
+MY_NAMESPACE=
+kubectl --context=$MY_CONTEXT -n $MY_NAMESPACE create cm ssh-key --from-file=authorized_keys=${HOME}/.ssh/id_rsa.pub --dry-run -o yaml | kubectl apply -f -
+kubectl --context=$MY_CONTEXT -n $MY_NAMESPACE apply -f https://raw.githubusercontent.com/armory/docker-debugging-tools/master/deployment-sshd.yml
+```
+To run the reverse proxy:
+```
+PORT_TO_FORWARD=
+kubectl --context=$MY_CONTEXT -n $MY_NAMESPACE port-forward deployment/sshd 2222:22 &
```

Comment:
This won't work trying to forward 2 ports because 2222 will be used locally. 

Add a newline between the `sleep` and `ssh -p...` and maybe a comment on forwarding multiple ports?

Reply:
Actually this works for multiple ports, `kubectl port-forward` is only used to be able to ssh login to the pod, but the actual forward is by using several `ssh -R...` commands. I'll add that to the README